### PR TITLE
Minor UI Change for /profile route.

### PR DESCRIPTION
**Pull request by yogeshbhutkar**
Issue addressed: #66 

What was the issue?
- The text was too faint to be visible.
- There wasn't a password visibility toggle already available.

Changes Made in this PR :
- Increased the color tone of the text to make it more visible.
- Increased the color tone of cancel's background to separate it from the background of UI a bit.
- Added a password visibility toggle.
- Made the text on the page like (Edit Profile) un-selectable to simulate professionalism.

A Screenshot of the implemented changes in this PR has been attached below :

image

### CODE REVIEW

File: src/app/(main)/profile/page.tsx [hunk 1]
```diff
@@ -73,36 +77,78 @@ export default function Page() {
...
-                  value={formData.password}
-                  onChange={(event) =>
-                    setFormData({
-                      ...formData,
-                      password: event.target.value,
-                    })
-                  }
-                />
+                <div className="relative z-0">
+                  <Input
```
Comment (skks1212) [inline src/app/(main)/profile/page.tsx:84]: The `Input` component has a `right` property. Use that for the show password button.

File: src/app/(main)/profile/page.tsx [hunk 2]
```diff
@@ -73,36 +77,78 @@ export default function Page() {
...
+                      strokeWidth={1.5}
+                      stroke="currentColor"
+                      className="w-5 h-5 text-gray-700"
+                    >
+                      <path
+                        strokeLinecap="round"
+                        strokeLinejoin="round"
+                        d="M3.98 8.223A10.477 10.477 0 001.934 12C3.226 16.338 7.244 19.5 12 19.5c.993 0 1.953-.138 2.863-.395M6.228 6.228A10.45 10.45 0 0112 4.5c4.756 0 8.773 3.162 10.065 7.498a10.523 10.523 0 01-4.293 5.774M6.228 6.228L3 3m3.228 3.228l3.65 3.65m7.894 7.894L21 21m-3.228-3.228l-3.65-3.65m0 0a3 3 0 10-4.243-4.243m4.242 4.242L9.88 9.88"
+                      />
+                    </svg>
```
Comment (skks1212) [inline src/app/(main)/profile/page.tsx:112]: Use Font Awesome!! No SVGs

**Review by skks1212 — CHANGES_REQUESTED** [review]
I don't think we need non selectable text. It feels very odd as it is not implemented anywhere else. Use font awesome instead of SVGs, and once the password is toggled change the icon to just the eye.

File: src/app/(main)/profile/page.tsx (hunk 1 above)
Comment (yogeshbhutkar) [reply src/app/(main)/profile/page.tsx:84]: Had a hard time using relative and z-axis, looks like there exists a simpler way. Thanks.

File: src/app/(main)/profile/page.tsx [hunk 3]
```diff
@@ -76,44 +80,68 @@ export default function Page() {
...
-                  }
-                />
+                <div className="relative z-0">
+                  <Input
+                    right={
+                      <div
+                        className="bg-gray-200 hover:bg-gray-300 px-4 py-2 rounded-md shadow-sm "
+                        onClick={() => setShowConfirmPassword((prev) => !prev)}
+                      >
+                        <FontAwesomeIcon
```
Comment (skks1212) [inline src/app/(main)/profile/page.tsx:116]: No need for the font awesome library! Look at icons implemented elsewhere in the codebase, a simple `<i className="far fa-eye"/>` is all you need 😀

**Review by skks1212 — CHANGES_REQUESTED** [review]
Please make the changes and pull the master branch

**Pull request closed without merging**